packages/flutter/lib/src/material/dialog.dart

```diff
@@ -2,11 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// TODO(shihaohong): remove ignoring deprecated member use analysis
```

Comment:
I moved the ignore to the deprecation notice so it can be removed with the deprecation symbol. When deprecations are removed, todos like these will probably be left behind.